src/assets/scripts/InputController.js

```diff
@@ -467,17 +481,17 @@ export default class InputController {
 
             case KEY_CODES.LEFT_ARROW:
                 // shortKeys in use
-                if (window.gameController.game.option.get('controlMethod') === 'arrows') {
-                    this.$commandInput.val(`${currentCommandInputValue} \u2BA2`);
+                if (this._isArrowControlMethod()) {
+                    this.$commandInput.val(`${currentCommandInputValue}t l `);
```

Comment:
we really need a way to make this consistent, and you're exactly right.   I'm glad you're keeping an eye on things!